Code/Framework/AzToolsFramework/AzToolsFramework/API/EditorPythonConsoleBus.h

```diff
@@ -63,6 +63,9 @@ namespace AzToolsFramework
         //! Signal the Python handler to stop
         virtual bool StopPython(bool silenceWarnings = false) = 0;
 
+        //! Query for the Python active state
```

Comment:
Can you add a comment on what python being active means? I don't know if this means that Python is actively running logic, or just Python is loaded and ready to do something.

Reply:
This is to detect if the Python VM has been already created, so that?

Reply:
A comment to that effect would be helpful, yeah.